ui/src/components/homepage/GoodFriday.js

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import {
+    Flex, 
+    Button, 
+    Text,
+    VStack,
+    Stack,
+    Link,
+    SimpleGrid
+  } from '@chakra-ui/react';
+import { AiOutlineDesktop, AiOutlineInstagram } from 'react-icons/ai';
+import { GiPartyPopper } from 'react-icons/gi';
+import { useMediaQuery } from '@chakra-ui/media-query';
+
+const HeroSection = () => {
+  const [isNotSmallerScreen] = useMediaQuery('(min-width:648px)');
+  return (
+    <Flex
+    w={['100%']}
+    h='110vh'
```

Comment:
<img width="1680" alt="Screenshot 2022-04-08 at 21 50 35" src="https://user-images.githubusercontent.com/33679483/162449368-1360914d-71de-466a-9fab-4456847b866f.png">

this is too big. when i load the screen the buttons are out of my screen. maybe change this to `85vh`